converted-ethereum-tests.txt

```diff
@@ -23,6 +23,13 @@ EOFTests/efStack/backwards_rjumpi_variable_stack_.json
 EOFTests/efStack/backwards_rjumpv_.json
 EOFTests/efStack/backwards_rjumpv_variable_stack_.json
 EOFTests/efStack/jumpf_stack_overflow_.json
+EOFTests/efStack/jumpf_stack_overflow_variable_stack_.json
```

Comment:
prague and osaka coverage check is disabled.

Reply:
That's fine. We are just doing it for our own documentation.